L1Trigger/L1TTrackMatch/plugins/L1TkHTMissEmulatorProducer.cc

```diff
@@ -0,0 +1,272 @@
+
+/**\class L1TrackerHTMissEmulatorProducer L1TrackerHTMissEmulatorProducer.cc
+ L1Trigger/L1TTrackMatch/plugins/L1TrackerHTMissEmulatorProducer.cc
+ Description: Takes L1TTkJets and performs a integer emulation of Track-based missing HT, outputting a collection of EtSum 
+*/
+
+// Original Author:  Hardik Routray
+//         Created:  Mon, 11 Oct 2021
+
+// system include files
+#include <memory>
+#include <numeric>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DataFormats/Math/interface/LorentzVector.h"
+#include "DataFormats/L1TCorrelator/interface/TkPrimaryVertex.h"
```

Comment:
I mean the fix to this line @BenjaminRS 
Thanks.

Reply:
To be honest I do not see where the vertex would be used in this class. I will double check with the author first, but I think it can be dropped altogether.

Reply:
Hi @srimanob -- I checked with the author of that file and he confirmed that it was a remnant of an older part of the code and that include can be dropped.